pagopa/io-app
# feat: [IOCOM-270] FIMS cookie clearing when leaving the FIMS webview

## Short description
This PR removes the FIMS token when leaving the FIMS webview.

## List of changes proposed in this pull request
- Added the proper wrapping method to the cookie manager
- `FimsWebViewScreen` refactored in order to access both domain and FIMS cookie name, caching unchanged data and avoiding unnecessary re-renderings

## How to test
Using the `io-dev-api-server`, start a FIMS flow and check that the FIMS cookie is properly set
